backend/src/utils/config.py

```diff
@@ -55,14 +54,11 @@ def load_env(self):
             self.mistral_url = os.getenv("MISTRAL_URL")
             self.mistral_key = os.getenv("MISTRAL_KEY")
             self.mistral_model = os.getenv("MODEL")
-            self.openai_key  = os.getenv("OPENAI_KEY")
+            self.openai_key = os.getenv("OPENAI_KEY")
             self.neo4j_uri = os.getenv("NEO4J_URI", default_neo4j_uri)
             self.neo4j_user = os.getenv("NEO4J_USERNAME")
             self.neo4j_password = os.getenv("NEO4J_PASSWORD")
             self.files_directory = os.getenv("FILES_DIRECTORY", default_files_directory)
-            self.azure_storage_connection_string = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
-            self.azure_storage_container_name = os.getenv("AZURE_STORAGE_CONTAINER_NAME")
-            self.azure_initial_data_filename = os.getenv("AZURE_INITIAL_DATA_FILENAME")
```

Comment:
These can be removed from the `.env.example` and the new `KNOWLEDGE_GRAPH_AGENT_LLM` and `KNOWLEDGE_GRAPH_AGENT_MODEL` added.